src/formpack/utils/ordered_collection.py

```diff
@@ -76,8 +73,8 @@ def __missing__(self, key):
 
     def __reduce__(self):  # optional, for pickle support
         args = (self.default_factory,) if self.default_factory else tuple()
-        return self.__class__, args, None, None, iteritems(self)
+        return self.__class__, args, None, None, iter(self.items())
 
     def __repr__(self):  # optional
         return '%s(%r, %r)' % (self.__class__.__name__, self.default_factory,
```

Comment:
Should we have a separate issue for converting the Python 2 string formatting to Python 3 styling?

Reply:
@joshuaberetta. Do you mean `f-strings`? 
Since `f-strings` is a 3.6 feature and it's gonna be deprecated in 3 months-ish, I totally agree. 
KPI is running under Python 3.8 anyway.